mno_analysis_tools/compute_msg_difference_btwn_two_firebase_time_periods.py

```diff
@@ -68,29 +68,36 @@ def date_time_range(start, end, delta):
             msg_direction = msg.direction
             filtered_messages.append(msg)
     log.info(f"returning {len(filtered_messages)} messages")
+    filtered_messages_timestamps = [msg.sent_on for msg in filtered_messages]
```

Comment:
Can you add a log line which describes what's about to happen? Otherwise the logs print "returning n messages" then it looks like the script is hanging

Reply:
updated thanks.